NOTICE

```diff
@@ -21876,16 +21881,16 @@ Distribution Requirements. For any Software Deliverables you distribute, you mus
 add significant primary functionality to it in your programs;
 require distributors and external end users to agree to terms that protect it at least as much as this Agreement;
 display your valid copyright notice on your programs; and
-indemnify, defend, and hold harmless the OPC Foundation from any claims, including attorneys’ fees, related to the distribution or use of your programs.
+indemnify, defend, and hold harmless the OPC Foundation from any claims, including attorneys� fees, related to the distribution or use of your programs.
```

Comment:
Please review the special characters that ended up here, like �. In the `NOTICE Opc.Ua.txt` that was provided those look like `’`. Maybe an encoding problem from the viewer that this has been copied from ?

Reply:
Fixed, now this shows up correctly.